### PR TITLE
Update: Changelog

### PR DESCRIPTION
**Pull request by LordAro**
I imagine the changelog list needs some work

Other files:
* README.md - no date
* known-bugs.txt - already updated
* installer.nsi - already pointing at 1.9.0-beta1, will be updated after beta release
* AI/GS changelogs, rev.cpp.in - not until actual release

### CODE REVIEW

File: changelog.txt [hunk 1]
```diff
@@ -1,3 +1,74 @@
...
+- Change: [NewGRF] Use last OpenTTD SVN revision in NewGRF version number (#6843)
+- Change: Clone tool in depot window now behaves like clone button in vehicle window (#6754)
+- Change #6397: Keep town growth rate in sync with house count
+- Change: Scale default FreeType font size selection by UI zoom level
+- Change: [OSX] Reversed pinch to zoom behaviour
+- Change: Switch town growth rate and counter to actual game ticks (#6763)
+- Change: Non-tag revisions are now named '<commitdate>-<branch>-g<shorthash><modified>'
+- Doc: Rewrite several bits of documentation in markdown
+- Fix #6584: Text layout in engine preview dialogue windows was
+- Fix #6636: Airplanes could be sent toto helicopter station depots
```
Comment (glx22) [inline changelog.txt]: toto ;)

**Review by nielsmh — DISMISSED** [review]
Looks good to me.

File: changelog.txt [hunk 2]
```diff
@@ -1,3 +1,75 @@
...
+- Change: [NewGRF] Use last OpenTTD SVN revision in NewGRF version number (#6843)
+- Change: Clone tool in depot window now behaves like clone button in vehicle window (#6754)
+- Change #6397: Keep town growth rate in sync with house count
+- Change: Scale default FreeType font size selection by UI zoom level
+- Change: [OSX] Reversed pinch to zoom behaviour
+- Change: Switch town growth rate and counter to actual game ticks (#6763)
+- Change: Non-tag revisions are now named '<commitdate>-<branch>-g<shorthash><modified>'
+- Change: Rewrite several bits of documentation in markdown
+- Fix: Potential crash during shutdown after unpacking downloaded content (#7185)
+- Fix #6584: Text layout in engine preview dialogue windows was
```
Comment (nikolas) [inline changelog.txt]: Incomplete sentence here: I would just remove "windows was" at the end, so it just reads "Fix #6584: Text layout in engine preview dialogue"

**Review by TrueBrain — APPROVED** [review]
Approving because @nielsmh  already did :D

File: changelog.txt [hunk 3]
```diff
@@ -1,3 +1,75 @@
...
+- Fix #7001: Pathfinders should see standard road stations as tiles where to reverse
+- Fix #6676: Prevent helicopters from stopping in midair during some kinds of landing
+- Fix: Remove need to use Ctrl+Click when building stations next to competitors (#6906)
+- Fix: [NewGRF] Make VA2 operator 11 (ror) behave well-defined when rotating by 0 bits
+- Fix #4109: Add more water checks to the Oil Rig layout
+- Fix #6938: Incorrect value for "Cost to Clear" was displayed for Company Headquarters in tile inspector (#6939)
+- Fix #6920: Make 9.8m/s^2 a common constant for TE-calculation
+- Fix #6892: [Script] CONFIG_RANDOM did not use the full parameter range (#6902)
+- Fix #6622: News message when GS constructs a town had empty company name
+- Fix: [NewGRF] Variable 0x85 had no bounds checks
```
Comment (frosch123) [inline changelog.txt]: - Fix: [NewGRF] Action7 variable 0x85 had no bounds checks

**Review by TrueBrain — CHANGES_REQUESTED** [review]
I take that back :P

Please also update `known-bugs.txt` for 1.9.0-beta1 (I think?)
Please update the date to today
Please look at your commit message :)

Tnx!

**Review by TrueBrain — APPROVED** [review]

**Pull request merged**